packages/definitions-parser/test/get-affected-packages.test.ts

```diff
@@ -55,9 +57,22 @@ testo({
       `/dt/types/has-older-test-dependency
 /dt/types/known`,
     ];
-    const { packageNames } = getAffectedPackagesWorker(allPackages, packageOutput, dependentOutput, "/dt");
+    const { packageNames } = getAffectedPackagesWorker(allPackages, packageOutput, [], dependentOutput, "/dt");
     expect(packageNames).toEqual(new Set(["jquery"]));
   },
+  newPackage() {
+    const packageOutput = ``;
+    const dependentOutput = ``;
+    const { packageNames, dependents } = getAffectedPackagesWorker(
+      AllPackages.from({ ...typesData, mistake: createTypingsVersionRaw("mistake", {}, {}) }, notNeeded),
```

Comment:
Should there be a test that only adds a file? This seems like it's creating a typings which I would think wouldn't exist? Maybe I'm not understanding.

Reply:
oops, that's left over from my misunderstanding of how package parsing works. In fact new, malformed packages do *not* show up in AllPackages, so the test is wrong--it should use the same mock data that all the other tests use.